frontend/src/components/WebhookManager.css

```
@@ -103,154 +120,124 @@
-.btn-delete:hover:not(:disabled) {
-  background-color: #b91c1c;
-}
-
-.btn-delete:disabled {
-  background-color: #9ca3af;
-  cursor: not-allowed;
-}
+/* Remove legacy button styles - using Rules section styles instead */
 
 .webhooks-list {
   space-y: 16px;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)